pytransform3d/rotations/_conversions.py

```diff
@@ -1383,6 +1383,122 @@ def extrinsic_euler_zxy_from_active_matrix(R, strict_check=True):
         R, unity, unitx, unitz, False, strict_check)[::-1]
 
 
+def euler_from_quaternion(q, i, j, k, extrinsic):
+    """General method to extract any Euler angles from quaternions.
+
+    Parameters
+    ----------
+    q : array-like, shape (4,)
+        Unit quaternion to represent rotation: (w, x, y, z)
+
+    i : int
+        Axis of the first rotation angle. A number between 0 and 2.
```

Comment:
Aren't axes more usually defined as numbers between `1` and `3`?

Reply:
This is consistent with [matrix_from_angle](https://dfki-ric.github.io/pytransform3d/_apidoc/pytransform3d.rotations.active_matrix_from_angle.html#pytransform3d.rotations.active_matrix_from_angle).